crowdfunding_compassion/controllers/main.py

```diff
@@ -24,8 +24,12 @@ def my_account(self, form_id=None, **kw):
                 ("project_id.project_owner_id", "!=", partner.id),
             ]
         )
-        donations = participations.mapped("invoice_line_ids").filtered(
-            lambda l: l.state == "paid")
+        donations = request.env["account.invoice.line"].sudo().search(
+            [
+                ("crowdfunding_participant_id.partner_id", "=", partner.id),
```

Comment:
You can directly filter in the search instead of doing 2 pass.
```suggestion
                ("crowdfunding_participant_id.partner_id", "=", partner.id),
                ("state", "=", "paid"),
```

Reply:
This is fixed by the next commit.